### PR TITLE
Build and render the DMD man page

### PR DESCRIPTION
**Pull request by wilzbach**
Needed for https://github.com/dlang/dmd/pull/7637 and https://github.com/dlang/dmd/pull/7648

### CODE REVIEW

File: posix.mak [hunk 1]
```diff
@@ -964,4 +966,25 @@ $G/contributors_list.ddoc:  | $(STABLE_RDMD) $(TOOLS_DIR) $(INSTALLER_DIR)
...
+	${MAKE} -C $(DMD_DIR)/docs DMD=$(abspath $(STABLE_DMD)) build
+	# Remove the loop after https://github.com/dlang/dmd/pull/7637 has been merged
+	for file in "$(DMD_DIR)/generated/$(MAN_PAGE)" "$(DMD_DIR)/$(MAN_PAGE)" ; do \
+		if [ -f "$$file" ] ; then \
+			mkdir -p $(dir $@); \
+			cp $$file $@; \
+			break; \
+		fi \
+	done
+	${MAKE} -s -C $(DMD_DIR)/docs DMD=$(abspath $(STABLE_DMD)) preview > $(dir $@)dmd.text
```
Comment (wilzbach) [inline posix.mak]: Maybe there's a better extension for such plaintext files?

Comment (JinShil) [reply posix.mak]: I've become quite accustomed to `.txt`

**Comment by wilzbach** [comment]
CC @CyberShadow - are you okay with this?

**Comment by CyberShadow** [comment]
Yep, the dlang.org part looks fine to me.

**Pull request merged**